engine/execution/computation/execution_verification_test.go

```diff
@@ -197,10 +197,6 @@ func executeBlockAndVerify(t *testing.T, txs [][]*flow.TransactionBody) *executi
 
 	for i, chunk := range er.Chunks {
 		isSystemChunk := i == er.Chunks.Len()-1
-		var collection flow.Collection
-		if !isSystemChunk {
-			collection = executableBlock.CompleteCollections[chdps[i].CollectionID].Collection()
-		}
```

Comment:
Collection is now part of the chunk data pack, which is prepared by the `ExecutionState, no need for a separate fetch.